ssh-server/main.go

```diff
@@ -118,16 +121,29 @@ func serviceChannel(newChannel ssh.NewChannel) error {
 	}
 	defer channel.Close()
 
-	allow := map[string]bool{"shell": true, "pty-req": true, "window-change": true}
+	allow := map[string]bool{"shell": true, "exec": true, "pty-req": true, "window-change": true}
 	resize := make(chan *pty.Winsize, 1)
 	defer close(resize)
 	done := make(chan error)
 	for {
 		select {
 		case err := <-done:
+			var status uint32
 			if err != nil {
-				return fmt.Errorf("running pty: %v", err)
+				if parts := exitStatusRegex.FindStringSubmatch(err.Error()); parts != nil {
```

Comment:
This is not the right way to recover the exit status... see os/exec's ExitError and ProcessState (or for an example, https://github.com/googlegenomics/pipelines-tools/blob/master/io/retry/retry.go)

Reply:
done